app/helpers/application_helper.rb

```diff
@@ -113,6 +113,24 @@ def fa_icon(icon, attributes = {})
     content_tag(:i, nil, attributes.merge(class: class_names.join(' ')))
   end
 
+  def material_symbol(icon, attributes = {})
+    return awesome_icon(icon.tr('_', '-'), attributes) if current_flavour == 'polyam'
+
+    inline_svg_tag(
+      "400-24px/#{icon}.svg",
+      class: %w(icon).concat(attributes[:class].to_s.split),
+      role: :img
+    )
+  end
+
+  def awesome_icon(icon, attributes = {})
+    inline_svg_tag(
+      "solid/#{icon}.svg",
+      class: %w(icon).concat(attributes[:class].to_s.split),
+      role: :img
+    )
+  end
```

Comment:
I'm not sure how to deal with regular icons yet.
Relying on upstream's `-fill` in filenames is not really an option.

Reply:
Dealt with by allowing `variant` attribute